### PR TITLE
React suggestions

### PR DESCRIPTION
**Pull request by johnthepink**
Hey man! Really nice work here. I put together a couple suggestions on initial look. I broke them up in to individual commits, so if you click in to each commit you can see a more thorough description.

Feel free to use any or none of this. <3

### CODE REVIEW

File: pwnedhub/static/react/messages.js [hunk 1]
```diff
@@ -143,14 +143,16 @@ var MessageDelete = React.createClass({
 });
 
 var Message = React.createClass({
-    render() {
+    getMessageStyle() {
         // set an inline style if the message is owned by the current user
-        var messageStyle = {};
         if (this.props.message.is_owner == true) {
```
Comment (lanmaster53) [inline pwnedhub/static/react/messages.js]: I elected to go with a ternary operator here.

`return this.props.message.is_owner == true ? {fontWeight: "bold"} : {}`

**Review by lanmaster53 — COMMENTED** [review]
I understand what the following code does, but what is its relevance to what I'm doing? Or was it just an example of the kind of stuff I could do if needed?

```
inputStyle = {
  float: isTrue ? "right" : "left"
}
```

File: pwnedhub/static/react/messages.js (hunk 1 above)
Comment (johnthepink) [reply pwnedhub/static/react/messages.js]: Even better! I love the ternary.

**Comment by johnthepink** [comment]
@lanmaster53 you're right it's not relevant. It was just an example of how you could inline an if statement using the ternary. Sorry that was confusing.

**Comment by lanmaster53** [comment]
Just realized this was still open. Closing. Thanks John!

**Pull request closed without merging**